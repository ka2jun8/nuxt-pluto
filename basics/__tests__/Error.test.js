import { shallowMount } from "@vue/test-utils";
import ErrorComponent from "../Error.vue";

describe("Error.vue", () => {
  const propsData = {
    error: "this is error test",
  };
  it("props", () => {
    const wrapper = shallowMount(ErrorComponent, { propsData });
    expect(wrapper.props()).toEqual(propsData);
  });
  it("snapshot", () => {
    const wrapper = shallowMount(ErrorComponent, { propsData });
    expect(wrapper.vm.$el).toMatchSnapshot();
  });
});
