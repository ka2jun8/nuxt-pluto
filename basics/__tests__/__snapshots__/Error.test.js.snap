// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error.vue snapshot 1`] = `
<p
  class="error"
>
  
  this is error test

</p>
`;
